web-app/src/app/app.component.css

```diff
@@ -69,11 +73,17 @@ and (max-width: 1100px){
   .tabs{
     margin-right: 3em;
   }
+  app-timestamp {
+    display: none;
+  }
 }
 @media only screen
 and (min-width: 1100px)
 and (max-width: 1200px){
   .tabs{
     margin-right: 2em;
   }
+  app-timestamp {
```

Comment:
Instead of having this block twice, can you do something like:
```
@media only screen
(min-width: 992px){
  app-timestamp {
    display: none;
  }
}
```